pose-detection/src/calculators/one_euro_filter.ts

```diff
@@ -77,8 +83,12 @@ export class OneEuroFilter {
         this.dx.applyWithAlpha(dValue, this.getAlpha(this.derivateCutOff));
```

Comment:
Quick question: the update rate is constant here in fact? Is that correct?
In our filter, we use an update rate of 0.5 here. Does that correspond to this.getAlpha(this.derivateCutOff)?